File: CastIt.Domain/Dtos/Responses/ServerPlayerStatusResponseDto.cs
using System.Collections.Generic;

namespace CastIt.Domain.Dtos.Responses
{
    public class ServerPlayerStatusResponseDto
    {
        public PlayerStatusResponseDto Player { get; set; }
        public GetAllPlayListResponseDto PlayList { get; set; }
        public FileItemResponseDto PlayedFile { get; set; }
        public List<FileThumbnailRangeResponseDto> ThumbnailRanges { get; set; } = new List<FileThumbnailRangeResponseDto>();
    }
}

File: CastIt.Client/ios/CastIt Watch App/Models/Responses/ServerPlayerStatusResponseDto.swift
import Foundation

// Mirrors ClientApp IServerPlayerStatusResponseDto
struct ServerPlayerStatusResponseDto: Codable {
    let player: PlayerStatusResponseDto
    let playList: GetAllPlayListResponseDto?
    let playedFile: FileItemResponseDto?
    let thumbnailRanges: [FileThumbnailRangeResponseDto]
}
